geniza/corpus/models.py

```diff
@@ -886,3 +886,11 @@ def __str__(self):
 
     def thumbnail(self):
         return self.fragment.iiif_thumbnails()
+
+
+class DocumentNeedsReview(Document):
+    """Proxy model for Document to give a separate view for documents that need review"""
+
+    class Meta:
+        proxy = True
+        verbose_name_plural = "Documents needing review"
```

Comment:
Will django let us put this in the admin instead of models? We really only want to use it there. I agree, it does seem annoying that we need a migration for a proxy model.

Wonder if we can shorten the name / label any. `ReviewDocuments` /  Documents for Review?

Reply:
I was able to move it to `admin.py` and renamed the model to "Documents for Review"